tests/stress.rs

```diff
@@ -1887,6 +1887,7 @@ unsafe extern "C" fn stress_functions(context: &dc_context_t) {
     buf1 = dc_decode_header_words(
         b"=?utf-8?B?dGVzdMOkw7bDvC50eHQ=?=\x00" as *const u8 as *const libc::c_char,
     );
+    println!("dc_decode_header_words {}", CStr::from_ptr(buf1 as *const libc::c_char).to_str().unwrap());
```

Comment:
On my machine this prints out:

![Screenshot from 2019-05-02 17-56-06](https://user-images.githubusercontent.com/308049/57088924-9b56d780-6d03-11e9-9d88-b5ae1a6675f1.png)

Reply:
This is what circle ci spits out (empty string?):

![Screenshot from 2019-05-02 18-03-16](https://user-images.githubusercontent.com/308049/57089375-9e05fc80-6d04-11e9-9592-c086629f0d47.png)

Reply:
So it seems `mailmime_encoded_phrase_parse` fails with error code 1:

![Screenshot from 2019-05-02 19-14-52](https://user-images.githubusercontent.com/308049/57093618-ac591600-6d0e-11e9-8160-6ee246d74df4.png)